src/components/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,22 @@
+import styles from './ProgressBar.module.css';
+
+function ProgressBar(props) {
+  const {humidityPct, progressBarHeight} = {...props}
```

Comment:
plz don't abbreviate
the height of `ProgressBar` is related to styling, should be in css and can add to this component via the className prop



` = {...props}` ????